flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/table/FileSystemOutputFormatTest.java

```diff
@@ -70,6 +85,11 @@ private static Map<File, String> getFileContentByPath(java.nio.file.Path directo
         return contents;
     }
 
+    private static Map<File, String> getStagingFileContent(
```

Comment:
> What we want to make sure is that the class actually creates its own staging folder

I believe this is the point we want to verify after moving the generation of the staging folder to `FileSystemOutputFormat`.

I understand your concern. In fact, `PartitionWriterTest` and `FileSystemCommitterTest` have already covered the checks for written content. Therefore, the tests for `FileSystemOutputFormat` should indeed focus more on the creation/deletion of directories. If we were to check file content as well, it would resemble an IT case.

I guess the reason content is checked is that the test author believes `FileSystemOutputFormat` has the capability to write records (implemented through the `writeRecord` method) and commit files from the staging folder to the outputPath (achieved with `finalizeGlobal`). Thus, it tested the written content even before our modifications were made.

Reply:
> The reason I am stressing on that part is that I'd like to remove the FileSystemOutputFormat#getStagingFolder method.

Good point! Let me remove it.